docs/docs/cli.md

```diff
@@ -0,0 +1,9 @@
+# Command line interface
```

Comment:
If this is client focused, can we have this explain the newer `drand-client` binary instead of the legacy grpc-based fetches built into the main drand binary?

`make client` will build the `drand-client` binary

Reply:
Ooh, good plan 👍. This part is copy/pasted - I'll open an issue and send a separate PR to change this.

Reply:
https://github.com/drand/website/issues/50